controllers/src/main/scala/cromwell/pipeline/controller/UserController.scala

```diff
@@ -42,13 +44,19 @@ class UserController(userService: UserService)(implicit executionContext: Execut
           }
         },
         put {
-          entity(as[PasswordUpdateRequest]) { passwordUpdateRequest =>
-            onComplete(userService.updatePassword(accessToken.userId, passwordUpdateRequest)) {
-              case Success(_)   => complete(StatusCodes.NoContent)
-              case Failure(exc) => complete(StatusCodes.BadRequest, exc.getMessage)
-            }
+          entity(as[PasswordUpdateRequest]) {
+            passwordUpdateRequest =>
+              FormValidatorNel.validateForm(passwordUpdateRequest) match {
+                case Valid(_) =>
+                  onComplete(userService.updatePassword(accessToken.userId, passwordUpdateRequest)) {
+                    case Success(_)   => complete(StatusCodes.NoContent)
+                    case Failure(exc) => complete(StatusCodes.BadRequest, exc.getMessage)
+                  }
+                case Invalid(errors) =>
+                  complete(StatusCodes.BadRequest -> errors.toList.map(_.toMap))
```

Comment:
We love Scala, because it gives us a lot of ways to do the same thing. But it would be nice to have unified syntax.  
`case Failure(exc) => complete(StatusCodes.BadRequest, exc.getMessage)`
`complete(StatusCodes.BadRequest -> errors.toList.map(_.toMap))`

Reply:
Should I change it?

Reply:
Yes, please